frontend/src/__mocks__/mockDscStatus.ts

```diff
@@ -134,7 +137,7 @@ export const mockDscStatus = ({
   installedComponents: Object.values(StackComponent).reduce(
     (acc, component) => ({
       ...acc,
-      [component]: installedComponents?.[component] ?? false,
+      [component]: installedComponents[component] ?? true,
```

Comment:
This should not default to true, it should stay as it was. The only change to this file should be the default value you added on line 36, if the user passes in installedComponents and that default is not used we shouldn't default everything they excluded to true.